apps/frontend/src/store/report_intake.ts

```diff
@@ -21,6 +21,7 @@ import {
   SnykResults,
   TwistlockMapper,
   XCCDFResultsMapper,
+  VeracodeMapper,
```

Comment:
there was a change in behavior that I believe got merged where the fingerprinting is now in hdfconverters instead of here in heimdall proper.  reach out to @Rlin232 for information on this.